### DISCUSSION TITLE
HELP : can't run after install on MAC M1

### DISCUSSION THREAD

**Discussion by ghost** [oobabooga/text-generation-webui#1565] [General]
Hi, I successfully managed to install with "sh start_macos.sh" but conda seems to fail to add its path to the $PATH.
After have added the path manually, I managed to run conda but it says it should be initialized, but "conda init bash" does nothing, and I can't launch textgen (it says conda is not initialized for textgen)

**Comment by ghost** [comment]
I somewhat managed to init conda with "conda init --all" but I get this error :


```
(textgen) ***@MacBook-Air text-generation-webui % python server.py
Traceback (most recent call last):
  File "/Users/***/LLaMA/ooba/text-generation-webui/server.py", line 2, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

**Comment by Kiru6ik** [comment]
try manual download. I faced difficulties running start_macos.sh before I downloaded miniconda, so try downloading it beforehand. But I am still facing difficulties on my M1 because it is not running on GPU due to OverflowError #712
